classes/UpgradeTools/CacheCleaner.php

```diff
@@ -75,7 +75,7 @@ public function cleanFolders(): void
                 if ($file[0] === '.' || $file === 'index.php') {
                     continue;
                 }
-                $this->container->getFilesystemAdapter()->clearDirectory($dir . $file, true);
+                $this->container->getFilesystemAdapter()->clearDirectory($dir . $file);
                 $this->logger->debug($this->container->getTranslator()->trans('File %s removed', [$file]));
```

Comment:
If you wanted to backport the changes from https://github.com/PrestaShop/autoupgrade/pull/1180, does it lack a part?